data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/leadership/leadership_max_value.cfg

```diff
@@ -4,13 +4,14 @@
 # API(s) being tested: [leadership]max_value=
 ##
 # Actions:
-# Give all units 50% self-leadership bonus damage with a max_value=50
+# Give all units 50% self-leadership bonus damage with a max_value=10
 # Attack each other
 ##
 # Expected end state:
 # max_value does nothing
 # The damage from the attack is increased by 50%
 #####
+#ifndef SCHEMA_VALIDATION
```

Comment:
max_value doesn't need this

Reply:
Sorry, I was wrong here - thought my test script had already check this while it was still running through campaigns.

Reply:
Schema validation fails otherwise:
`error validation: Invalid key 'max_value=' in tag [leadership]`